market/connector/google-geo-api/google-geo-api-demo/webContent/layouts/js/map-dialog-handler.js

```diff
@@ -0,0 +1,40 @@
+function initMap() {
```

Comment:
I guess it would fit better to move this .js to the HtmlDialog that uses it:

![hdResourcesDir](https://user-images.githubusercontent.com/15085693/115834371-73dc0680-a415-11eb-9735-34edf9734849.png)

Reply:
Just browsed over this PR again. I think we do not support yet to automatically add contents of the Html Dialog 'resources' directory via project-resource-installer/dialogs ... so just be warned, that you'll have to copy these resources manually into the target directory with the simple 'copy-resource' approach

Reply:
- [x] Done!

Reply:
Good to have the js files closer to the Dialog. But please place it in the new 'Viewer' dialog. Otherwise the component will not function independently. Users can then delete the demo parts of the connector, once they have examined it. If the featureset is spread over multiple dialogs, it can't be easily taken apart.
![allJSCodeToItsViewer](https://user-images.githubusercontent.com/15085693/116097511-7efb9480-a6aa-11eb-8a03-d1e3f6ccb3b7.png)